README.md

```diff
@@ -7,71 +7,69 @@
 
+<br><br>
 
 **For detailed documentation** about Getting Started with RPS or other features of the Open AMT Cloud Toolkit, see the [docs](https://open-amt-cloud-toolkit.github.io/docs/).
 
+<br>
+
 ## Prerequisites
 
 To succesfully deploy RPS, the following software must be installed on your development system:
 
 - [Node.js* LTS 12.x.x or newer](https://nodejs.org/en/)
 - [git](https://git-scm.com/downloads)
-- [Management Presense Server](https://github.com/open-amt-cloud-toolkit/mps)
 
 
 ## Deploy the Remote Provisioning Server (RPS) Microservice
 
 To deploy the RPS on a local development system: 
 
-1. For a local setup, RPS requires an [installed MPS](https://github.com/open-amt-cloud-toolkit/mps) in the same parent directory. Clone the RPS repository to the same parent directory where your mps directory is located. 
-
-    ```
-    📦parent
-     ┣ 📂mps
-     ┗ 📂rps
-    ```
+1. Clone the repo and switch to the `rps` directory.
 
     ```
     git clone https://github.com/open-amt-cloud-toolkit/rps.git && cd rps
     ```
 
-2. Run 'npm install' from the working rps directory.
+2. Install the dependencies from the working `rps` directory.
 
     ``` bash
     npm install
     ```
 
-3. Run 'npm run dev' start command. The npm run dev start command may take 1-2 minutes to install.
+3. Start the service.
 
     ``` bash
-    npm run dev
+    npm start
     ```
-    
-    >Note: Warning messages are okay and expected for optional dependencies.
 
-4. The RPS listens on port 8081. Successful installation produces the command line message:
+4. The RPS listens on port 8081 by default. Successful installation produces the command line message:
 
     ```
-    WebSocketListener - RPS Microservice socket listening on port: 8080 ...!
     RPS Microservice Rest APIs listening on https://:8081.
     ```
     
 For detailed documentation about RPS, see the [docs](https://open-amt-cloud-toolkit.github.io/docs/)
 
+<br>
 
 ## Running the API Tests
 
 We leverage [Postman](https://www.postman.com/) and Docker for executing RESTful API tests. Once you have Postman and Docker installed, you can follow the steps below:
 
-1) Build the docker image with the following command:
-`docker build -t rps-microservice:v1 .`
+1. Build the docker image.
+    ```
+    docker build -t rps-microservice:v1 .
+    ```
 
-2) Ensure RPS is running in a docker container with the following command:
-`docker run -d -e RPS_HTTPS=false -e RPS_NODE_ENV=dev -e RPS_USE_VAULT=false  -e RPS_USE_DB_PROFILES=false  -e RPS_LOG_LEVEL=silly -p 8081:8081 rps-microservice:v1`
+2. Ensure RPS is running in a docker container.
+    ```
+    docker run -d -e RPS_HTTPS=false -e RPS_NODE_ENV=dev -e RPS_USE_VAULT=false  -e RPS_USE_DB_PROFILES=false -e RPS_LOG_LEVEL=silly -p 8081:8081 rps-microservice:v1`
```

Comment:
did you try this? I don't think this actually works as those ENV vars are outdated. Should be `docker-compose up -d`

Reply:
Updated. However, is this section still valid anymore? The Postman collection still seems to still reflect the old APIs with an API Key.  Should this section be removed overall or should collection be updated?

Reply:
that collection is what is run everyday in CI/CD, so it might have remnants but it should be up to date.